jetty-ee10/jetty-ee10-webapp/src/main/config/etc/jetty-ee10-deploy.xml

```diff
@@ -35,13 +35,13 @@
           <Set name="defaultsDescriptor">
             <Property>
               <Name>jetty.deploy.defaultsDescriptorPath</Name>
+              <Deprecated>jetty.deploy.defaultsDescriptor</Deprecated>
               <Default>
-                <Property name="jetty.home" default="." />/etc/webdefault-ee10.xml
+                <Property name="jetty.base" default="." />/etc/webdefault-ee10.xml
```

Comment:
This one is odd. (and was duplicated below in same way for ee9/ee8 too)
The `<Set name="defaultDescriptor" ...` was declared twice in this XML.
It felt correct to remove the second call, but I'm not 100% sure that's the right decision here.
Also note the difference between `jetty.base` and `jetty.home` above.